src/LuauExt.cpp

```diff
@@ -624,7 +712,8 @@ struct FindSymbolReferences : public Luau::AstVisitor
         {
             if (visitLocal(var))
             {
-                result.push_back(var->location);
+                addResult(var->location, lsp::DocumentHighlightKind::Write);
```

Comment:
Yep, it is the i,v. I think all loop variables should be write, because it's basically declaring a variable (similar to `local i, v = fun(...)`)? Sumneko lua has them be write too.